ansible/roles/schulcloud-server-core/templates/admin-api-server-deployment.yml.j2

```
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: admin-api-deployment
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: admin-api-serv
+    app.kubernetes.io/part-of: schulcloud-verbund
+    app.kubernetes.io/version: {{ SCHULCLOUD_SERVER_IMAGE_TAG }}
+    app.kubernetes.io/name: admin-api-serv
+    app.kubernetes.io/component: admin-api
```

It is part of the server so the componet is server in this case
```suggestion
    app.kubernetes.io/component: server
```

---

done